panther_manager/src/manager_bt_node.cpp

```diff
@@ -137,6 +141,12 @@ ManagerBTNode::ManagerBTNode(
 
   if (launch_safety_tree) {
     const std::map<std::string, std::any> safety_initial_bb = {
+      {"aux_state", false},
+      {"bat_temp", std::numeric_limits<float>::quiet_NaN()},
+      {"cpu_temp", std::numeric_limits<float>::quiet_NaN()},
+      {"driver_temp", std::numeric_limits<float>::quiet_NaN()},
+      {"e_stop_state", true},
+      {"fan_state", true},
```

Comment:
These values are unused. During the timer loop execution, the first operation is to overwrite these values, which consequently renders them unused. In my opinion, this is a kind of unnecessary overzealousness. Furthermore, these values can be misleading since they do not always have such initial values (e.g., e_stop_state).

Reply:
I reverted those changes